.github/PULL_REQUEST_TEMPLATE/pull-request-template.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Pull Request
+about: Pull Request Template
+title: ''
+labels: ''
+assignees: ''
+---
+
+Thank you for your Pull Request!
+
+Before you submit, please check that:
+
+-   [ ] You mentioned a related Issue if this PR related to it, e.g. `Fixes #228` or
+        `Related #1337`.
+-   [ ] You added a brief description of the PR, e.g.:
+
+    -   What does it do?
+    -   What important points reviewers should know?
+
+-   [ ] You labeled the PR appropriately with the labels listed in the sidebar
+-   [ ] PR status, release notes, various implications/requirements
+-   [ ] You asked any particular reviewers to review. If you aren't sure, start with GH suggestions
+-   [ ] Your PR adheres to the style guide
+    -   In particular, mind the maximal line length of 100 (120 in exceptional circumstances)
+    -   `yarn lint` is run
+    -   There is no commented code checked in unless necessary
+-   [ ] You updated any documentation that may have changed
```

Comment:
Some of these have terminal punctuation and some do not. Make it consistent (I prefer with).